dist/scripts/api-console.js

```diff
@@ -1871,52 +1878,65 @@
         }
 
         $scope.hasResourcesWithChilds = function () {
-          return $scope.raml.resourceGroups.filter(function (el) {
+          return $scope.resourceGroups.filter(function (el) {
             return el.length > 1;
           }).length > 0;
         };
 
-        ramlParserWrapper.onParseError(function(error) {
-          $scope.error  = error;
+        ramlParserWrapper.onParseError(function(errors) {
           $scope.loaded = true;
 
-          /*jshint camelcase: false */
-          var context = error.context_mark || error.problem_mark;
-          /*jshint camelcase: true */
-
-          $scope.errorMessage = error.message;
+          $scope.errors = errors.parserErrors.map(function (theError) {
+            return {
+              errorMessage: theError.message,
+              line: theError.line,
+              column: theError.column
+            };
+          });
 
-          if (context) {
-            $scope.raml = context.buffer;
+          if (!$scope.loadFromUrl) {
```

Comment:
This is the second time I see this exact same block, we need to extract this to a helper function somewhere.